tailwind.config.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
   content: [
     ...tailwindConfigs.content,
     "./node_modules/@axodotdev/fringe/lib/*.vue",
+    "../turbo-axo/packages/fringe/lib/*.vue",
```

Comment:
**question:** I'm not sure I understand why we need this. We should be accessing Fringe using `yarn link` so that the `node_modules` reference suffices.

Reply:
mine doesnt work, I can remove it but ill neeed to add it to work with link

Reply:
I removed it. Let's pair you & me or you & @ashleygwilliams to see if we can't get your link working correctly.